src/apps/uclient/session.h

```diff
@@ -1,4 +1,6 @@
 /*
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I do not understand what this is - let's keep it out of the PR

Reply:
This is a software bill of materials tag that helps automated tools understanding the provenance of the code.

This is really important to have.

Reply:
Why should coturn repo add a random software's tag into its code?

Reply:
And in any case, let's not mix different changes in one PR
In this PR you change ints to bools

Reply:
> Why should coturn repo add a random software's tag into its code?

SPDX is a standard that's widely used in the open source software ecosystem, it's not a single specific piece of software by some specific vendor. It's an industry standard.

E.g. the Linux kernel uses it, among uncountable many others.

https://www.kernel.org/doc/html/v4.18/process/license-rules.html

Reply:
> And in any case, let's not mix different changes in one PR In this PR you change ints to bools

Sure, I can split the change.

Reply:
split